DocumentDataAPI/DocumentDataAPI/Relevance Function/CosineSimilarityCalculator.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DocumentDataAPI.Relevance_Function;
```

Comment:
Omdøb namespace til at være PascalCase (`RelevanceFunction`) - husk at omdøbe mappen, og gøre det samme for test-projektet!!!